resources/views/stats/clients/clients.blade.php

```diff
@@ -0,0 +1,37 @@
+@extends('layout.default')
+
+@section('title')
+    <title>@lang('stat.stats') - {{ config('other.title') }}</title>
+@endsection
+
+@section('breadcrumb')
+    <li class="active">
+        <a href="{{ route('stats') }}" itemprop="url" class="l-breadcrumb-item-link">
+            <span itemprop="title" class="l-breadcrumb-item-link-title">@lang('stat.stats')</span>
+        </a>
+    </li>
+    <li>
+        <a href="{{ route('languages') }}" itemprop="url" class="l-breadcrumb-item-link">
+            <span itemprop="title" class="l-breadcrumb-item-link-title">Clients</span>
+        </a>
+    </li>
+@endsection
+
+@section('content')
+    <div class="container">
+        <div class="block">
+            <h2>Clients</h2>
+            <hr>
+            <div class="row col-md-offset-1">
+                @foreach ($clients as $key => $value)
```

Comment:
maybe should add some default sorting for agent alphabetically?

Reply:
We could put sorting in `StatsController.php` to keep the blade file clean.

It seems, that PHP has a neat sorter for sorting by keys in an array: https://www.php.net/manual/en/function.ksort.php

However I am thinking about adding a string length limiter in the blade file, to avoid too long Agent strings, which might result in poor looking boxes.

Reply:
sounds good to me